### ISSUE TITLE
WINDOWS: Launching subshell, not parsing paths correctly

### ISSUE THREAD

**Issue by k33k00** [kennethreitz/pipenv#448]
I'll be honest I can't find a reference to this in your code so it might be someone elses problem (mine?)

I'm running [cmder](http://cmder.net/) on windows 7 when I do `pipenv shell` I get the following output

```
Launching subshell in virtual environment. Type 'exit' to return.
'C:\Program' is not recognized as an internal or external command,
operable program or batch file.
```

Reproduce
------------

- install [cmder](http://cmder.net/) into program files 
- launch one of your virtual environments with `pipenv shell` 
- keep a close eye on the output

My theory
-----------

My guess is, it is having problems with space in `C:\Program Files`. Which got me to thinking, why does it need program files anyway. I can only assume it's trying to call back to cmder to set up it's environment.

Anyway I'll download pipenv from the master branch later and have a proper poke around.

Tests
-----

- Installed cmder into `C:/tools` and it worked perfectly.
- tried it in a standard cmd prompt and it worked perfectly.

**Comment by k33k00** [comment]
I should add this does not stop anything from working apart from cmder's layout

**Comment by nateprewitt** [comment]
This sounds like it's definitely a path issue of some kind. virtualenv has issues with spaces in the path and I'm not entirely sure if the hangup is there, in pew, or in cmder itself. I don't have windows environment to properly test this in currently, but our general advice has been to avoid paths with spaces when possible.

**Comment by k33k00** [comment]
No worries then, Like I said it's not breaking anything major. I've moved cmder to C:/tools for now

**Comment by florisla** [comment]
The error message is produced by pew (Lib\site-packages\pew\pew.py).

See also #184, #237.

**Comment by techalchemy** [comment]
Oh, this may incidentally be fixed in #893
